worksheet_C/readme.MD

```diff
@@ -0,0 +1,10 @@
+Worksheet C  
+A) The algorithm checks if there is a duplicate letter in the list by going through the list in order to see if a letter is repeated, if a letter has a duplicate the result will be true. 
+B) Because of the loop within the loop, so it has to check each item in the first list as well as each item in the second list so it will be N * N = N^2 
+C) Because it still checks all the numbers but it doesn't check the last number because its been check already making it almost twice as fast. Eg : 1:2 1:1 2:2 2:1 
+D) Because the first algorithm has to check it 100 times because of N^2 where as this one only has to check one less number each loop which means it’s a lot faster. 
+E) No because the number of times is not N^2 anymore because there is one less number in the loop it has to check. 
```

Comment:
As explained in class, the algorithm is still quadratic -- there are now n*(n-1)/2 checks to do, which is still O(n^2).